### ISSUE TITLE
[Accessibility]: Screen reader is announcing inappropriate information when focus is on dropdown in edge browser

### ISSUE THREAD

**Issue by tabansal** [microsoft/fluentui#15250]
Dropdowns are adding an attribute aria-haspopup="listbox" to parent div in dropdowns. This cases screen readers to announce haspopup along with other information for drop downs in edge. Issue is reproducible on https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown with narrator.

Can someone please look into this?

Regards,
Tapas Bansal
alias: tabansal

**Comment by paulgildea** [comment]
@tabansal Thanks for the issue.

@smhigley FYI on the pattern.

**Comment by miroslavstastny** [comment]
I can confirm `aria-haspopup="listbox"` is always added:
https://github.com/microsoft/fluentui/blob/7fc8435387fa90c21cdc396b41ed43f6f9e7979b/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx#L305

@kolaps33 - what is the expected DOM/ARIA structure?

**Comment by tabansal** [comment]
Any updates team?

**Comment by tabansal** [comment]
@kolaps33 @miroslavstastny can you please prioritize this bug since it's impacting accessibility signoff for our project

**Comment by dcowen91** [comment]
Hi @miroslavstastny  - I just wanted to see if this issue is targeted for any upcoming milestones?

**Comment by smhigley** [comment]
Fixed by #17603

@tabansal let me know if you still run into any issues once the change is released (it's going into both 8 and 7). Dropdown has been updated to the ARIA 1.2 pattern, so that should solve the structural issues. Aria-haspopup="listbox" is actually a necessary attribute on the Dropdown pattern and should be announced by screen readers, but previously the element it was on had the wrong role, so the way it was announced was incorrect. That should be fixed by the linked PR :)
